eclipsecs-sevntu-plugin/src/com/github/sevntu/checkstyle/checks/design/checkstyle-metadata.properties

```diff
@@ -49,7 +49,7 @@ HideUtilityClassConstructorCheck.name = Hide Utility Class Constructor
 HideUtilityClassConstructorCheck.desc = Make sure that utility classes (classes that contain only static methods) do not have a public constructor.
 
 InnerClassCheck.name = Inner Class
-InnerClassCheck.desc = Check nested (internal) classes to be�declared�at the bottom of the class after all methods (fields) declaration.
+InnerClassCheck.desc = Check nested (internal) classes to be declared at the bottom of the class after all methods (fields) declaration.
```

Comment:
Somehow this file had hex 0xA0 bytes here. This was found when the test loading the file was changed slightly.